### PR TITLE
BZ1989855 Adding cluster-ingress-default-ingresscontroller.yaml step

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.8+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=1989855

Link to docs preview:
https://54222--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-vpc.html#nw-gcp-global-access-configuration_installing-gcp-vpc

QE review:
- [X] QE has approved this change.

### CODE REVIEW

**Comment by cbippley** [comment]
@jianli-wei Could you take a look at this? Thank you!

**Comment by kalexand-rh** [comment]
@jianli-wei, will you PTAL?

File: modules/nw-gcp-installing-global-access-configuration.adoc [hunk 1]
```diff
@@ -25,6 +25,15 @@ $ ./openshift-install create manifests --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the name of the directory that
 contains the `install-config.yaml` file for your cluster.
 +
+. Create a file that is named `cluster-ingress-default-ingresscontroller.yaml` in the `<installation_directory>/manifests/` directory:
++
+[source,terminal]
+----
+$ touch <installation_directory>/manifests/cluster-ingress-default-ingresscontroller.yaml <1>
```
Comment (jianli-wei) [inline modules/nw-gcp-installing-global-access-configuration.adoc:32]: As it's a newly created YAML file, the subsequent section `3. Open the cluster-ingress-default-ingresscontroller.yaml file in an editor...` should tell the complete content of a sample YAML file instead, for example, 
```
apiVersion: operator.openshift.io/v1
kind: IngressController
metadata:
  creationTimestamp: null
  name: default
  namespace: openshift-ingress-operator
spec:
  endpointPublishingStrategy:
    loadBalancer:
      providerParameters:
        gcp:
          clientAccess: Global 
        type: GCP
      scope: Internal          
    type: LoadBalancerService

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: modules/nw-gcp-installing-global-access-configuration.adoc (hunk 1 above)
Comment (jianli-wei) [reply modules/nw-gcp-installing-global-access-configuration.adoc:32]: @gpei Could you help review the content of the sample YAML too, I'm not quite sure it is correct, thanks!

Comment (gpei) [reply modules/nw-gcp-installing-global-access-configuration.adoc:32]: @jianli-wei seen from our flexy repo, the YAML content is correct.

`Changing the default ingress LoadBalancer with Global access enabled on GCP`
```
 cat <<EOF >manifests/cluster-ingress-default-ingresscontroller.yaml
apiVersion: operator.openshift.io/v1
kind: IngressController
metadata:
  name: default
  namespace: openshift-ingress-operator
spec:
  endpointPublishingStrategy:
    loadBalancer:
      scope: Internal
      providerParameters:
        type: GCP
        gcp:
          clientAccess: Global
    type: LoadBalancerService
EOF
```

**Review by jianli-wei — APPROVED** [review]

**Comment by bscott-rh** [comment]
LGTM.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**